src/Adapter/SymfonyContainer.php

```diff
@@ -47,10 +49,8 @@ final class SymfonyContainer
     public static function getInstance()
     {
         if (!isset(self::$instance)) {
-            global $kernel;
```

Comment:
To be marked inside the PR as a BC break (we _accept_ BC breaks in 8.0.x but we need to write them down)

Reply:
reverted